README.md

```diff
@@ -71,38 +82,125 @@ Add the following content to the `.env` file:
 # OpenAI API Key
 OPENAI_API_KEY=
 
-# Assistant ID - leave it empty if you don't have an assistant yet
+# Assistant ID - leave it empty if you don't have an assistant to reuse
 ASSISTANT_ID=
 ```
 
-Please note that the `.env` file should not be committed to the repository. Add it to the `.gitignore` file to prevent it from being committed.
+Please note that the `.env` file should not be committed to the repository. *Add the `.env` file to the `.gitignore`* file to prevent it from being committed.
 
 ### Step 3: Configuration
 
 The library provides a way to configure the assistant with the `AssistantModule.forRoot` method. The method takes a configuration object as an argument. Create a new configuration file like in a [sample configuration file (chat.config.ts)](apps%2Fapi%2Fsrc%2Fapp%2Fchat%2Fchat.config.ts) and fill it with the necessary configuration. 
 
-More details about the configuration with code examples can be found in the [wiki](https://github.com/boldare/openai-assistant/wiki/%F0%9F%A4%96-AI-Assistant#step-3-configuration).
+```typescript
+// chat.config.ts file
+import { AssistantConfigParams } from '@boldare/openai-assistant';
+import { AssistantCreateParams } from 'openai/resources/beta';
+
+// Default OpenAI configuration
+export const assistantParams: AssistantCreateParams = {
+  name: 'Your assistant name',
+  instructions: `You are a chatbot assistant. Speak briefly and clearly.`,
+  tools: [
+    { type: 'code_interpreter' }, 
+  ],
+  model: 'gpt-4-turbo',
+  temperature: 0.05,
+};
+
+// Additional configuration for our assistant
+export const assistantConfig: AssistantConfigParams = {
+  id: process.env['ASSISTANT_ID'],
+  params: assistantParams,
+  filesDir: './apps/api/src/app/knowledge',
+  toolResources: {
+    codeInterpreter: {
+      fileNames: [],
+    },
+  },
+};
+```
```

Comment:
I am not sure if we want to display here the configuration or link to the example would be enough. 🤔

Reply:
Unless we provide a code snippet, the reader(especially non technical one) may encounter a friction point. I perceive this as an opportunity to make it more popular throughout the non-technical persona users. This is also why I tried to make it as short as possible and threw the file search out as this takes a few lines.